src/components/game/table/useContextMenuPreventer.js

```diff
@@ -0,0 +1,15 @@
+import { useEffect } from 'react';
+
+export const useContextMenuPreventer = () => {
```

Comment:
O co chodzi z tym context menu?

Reply:
jak klikniesz prawym to pojawia sie context menu, ze mozesz zpisac obrazek itd, nie chcemy tego podczas gry, szczegolnie ze na prawym obraca sie klocka